### PR TITLE
adding two more plays to udracor

### PR DESCRIPTION
**Pull request by JuliaDudar**

### CODE REVIEW

**Comment by JuliaDudar** [comment]
added two plays:
kvitka-osnovianenko-hryhoriy-fedorovych-shelmenko;
kropyvnytskyy-marko-lukych-hlytay-abo-zh-pavuk

**Review by cmil — CHANGES_REQUESTED** [review]
Hi Julia, welcome to UDraCor. It's great to see a new contributor. Thank you for your pull request! The two new document look very good already. There are a few minor validation and DraCor specific issues. See the respective comments. Maybe we can iron those out before merging. Let me know if you need further support. Also @DanilSko would probably be able to help.

**Comment by JuliaDudar** [comment]
Hi Carsten, thank you for welcome words and your revision, sorry for the late answer. There are some quetions that I first want to discuss with  @DanilSko before finishing the files.

File: tei/kropyvnytskyy-marko-lukych-hlytay-abo-zh-pavuk.xml [hunk 1]
```diff
@@ -0,0 +1,2167 @@
...
+          </person>
+          <person sex="MALE" xml:id="bychok">
+            <persName>Бичок</persName>
+          </person>
+          <person sex="MALE" xml:id="odyn_z_parubkiv">
+            <persName>Один з парубків</persName>
+          </person>
+        </listPerson>
+      </particDesc>
+      <revisionDesc>
```
[the remark this answers is not in the document]
Comment (JuliaDudar) [reply tei/kropyvnytskyy-marko-lukych-hlytay-abo-zh-pavuk.xml]: done

File: tei/kvitka-osnovianenko-hryhoriy-fedorovych-shelmenko-dens.xml [hunk 2]
```diff
@@ -0,0 +1,3388 @@
...
+      </particDesc>
+    </profileDesc>
+    <revisionDesc>
+      <listChange>
+        <change when="2022-12-02">Julia Dudar encoded the source text</change>
+      </listChange>
+    </revisionDesc>
+  </teiHeader>
+  <text>
+    <body>﻿<castList>
```
[the remark this answers is not in the document]
Comment (JuliaDudar) [reply tei/kvitka-osnovianenko-hryhoriy-fedorovych-shelmenko-dens.xml:84]: done

File: tei/kropyvnytskyy-marko-lukych-hlytay-abo-zh-pavuk.xml [hunk 3]
```diff
@@ -0,0 +1,2174 @@
+<TEI xml:lang="ukr" xml:id="u0000033" xmlns="http://www.tei-c.org/ns/1.0">
+  <teiHeader>
+    <fileDesc>
+      <titleStmt>
+        <title type="main">Глитай, або ж павук</title>
+        <title type="sub">Драма у 4-х діях і 5-ти одмінах</title>
+        <author>
+          <persName>
+            <forename>Марко</forename>
+            <surnameКропивницький</surname>
```
Comment (cmil) [inline tei/kropyvnytskyy-marko-lukych-hlytay-abo-zh-pavuk.xml:10]: There is a missing `>` for the surname tag.

Comment (DanilSko) [reply tei/kropyvnytskyy-marko-lukych-hlytay-abo-zh-pavuk.xml:10]: Yes, thanks, we noticed that too) I suggested fix for that an a couple of other stuff here yesterday: https://github.com/JuliaDudar/udracor/pull/1

File: tei/kropyvnytskyy-marko-lukych-hlytay-abo-zh-pavuk.xml [hunk 4]
```diff
@@ -0,0 +1,2167 @@
...
+          <availability status="free">
+            <p>In the public domain.</p>
+          </availability>
+        </bibl>
+      </sourceDesc>
+    </fileDesc>
+    <profileDesc>
+      <particDesc>
+        <listPerson>
+          <person sex="FEMALE" xml:id="1-a_molodytsja">
```
[the remark this answers is not in the document]
Comment (JuliaDudar) [reply tei/kropyvnytskyy-marko-lukych-hlytay-abo-zh-pavuk.xml]: done

File: tei/kropyvnytskyy-marko-lukych-hlytay-abo-zh-pavuk.xml [hunk 5]
```diff
@@ -0,0 +1,2167 @@
+<TEI xmlns="http://www.tei-c.org/ns/1.0">
+  <teiHeader>
+    <fileDesc>
+      <titleStmt>
+        <title type="main">Глитай, або ж павук</title>
+        <title type="sub">Драма у 4-х діях і 5-ти одмінах</title>
+        <author>Марко Кропивницький</author>
```
[the remark this answers is not in the document]
Comment (JuliaDudar) [reply tei/kropyvnytskyy-marko-lukych-hlytay-abo-zh-pavuk.xml]: done

File: tei/kvitka-osnovianenko-hryhoriy-fedorovych-shelmenko-dens.xml [hunk 6]
```diff
@@ -0,0 +1,3388 @@
+<TEI xmlns="http://www.tei-c.org/ns/1.0">
+  <teiHeader>
+    <fileDesc>
+      <titleStmt>
+        <title type="main">Шельменко-денщик</title>
+        <title type="sub">Комедия в пяти действиях</title>
+        <author>Григорій Квітка-Основ'яненко</author>
```
[the remark this answers is not in the document]
Comment (JuliaDudar) [reply tei/kvitka-osnovianenko-hryhoriy-fedorovych-shelmenko-dens.xml]: done

**Comment by DanilSko** [comment]
@cmil um, Carsten, is it also possible to have the validation workflow run automatically without manual approval here?

**Comment by DanilSko** [comment]
Thank you, Carsten! I see that the tests now run for this PR 🙏
(and seem to pass now)

**Review by cmil — APPROVED** [review]

**Comment by cmil** [comment]
Thanks for making the changes @JuliaDudar and @DanilSko! I will merge now. :)

**Comment by DanilSko** [comment]
Thank you @cmil! and of course you, @JuliaDudar! Happy holidays!

**Comment by cmil** [comment]
Happy holidays to you too!

**Comment by JuliaDudar** [comment]
Thank you guys, wish you happy holidays as well!

**Pull request merged**